src/librustc_data_structures/graph/mod.rs

```diff
@@ -282,25 +251,6 @@ impl<N: Debug, E: Debug> Graph<N, E> {
         self.incoming_edges(target).sources()
     }
 
-    /// A common use for graphs in our compiler is to perform
-    /// fixed-point iteration. In this case, each edge represents a
-    /// constraint, and the nodes themselves are associated with
-    /// variables or other bitsets. This method facilitates such a
-    /// computation.
-    pub fn iterate_until_fixed_point<'a, F>(&'a self, mut op: F)
```

Comment:
I was pretty surprised that this is dead code. But as far as I can tell, #7688 introduced it alongside with another fixed-point iteration function definition ... and I guess no one ended up ever invoking the one that used the new graph abstraction ... Maybe that was meant to be future work that never got done? Anyway, I don't object to removing it now.